teams/teams-ps/teams/Get-CsTeamsShiftsConnectionConnector.md

```diff
@@ -0,0 +1,55 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Get-CsTeamsShiftsConnectionConnector
+author: gucsun
```

Comment:
author metadata seems ok however ms.author: metadata is missing, it should be your Microsoft alias.